conans/client/cmd/uploader.py

```diff
@@ -73,21 +75,42 @@ def __init__(self, cache, user_io, remote_manager, loader, hook_manager):
         self._remote_manager = remote_manager
         self._loader = loader
         self._hook_manager = hook_manager
+        self._num_threads = 1
```

Comment:
I would prefer to avoid these attributes, but pass instead things as arguments when necessary.